x-pack/plugins/lens/public/trigger_actions/open_in_discover_action.test.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import { DiscoverStart } from '@kbn/discover-plugin/public';
+import { DiscoverSetup } from '@kbn/discover-plugin/public';
```

Comment:
Not sure I understand this change. `createOpenInDiscoverAction` is [called during the plugin start phase](https://github.com/elastic/kibana/pull/131237/files#diff-2922e379d14ccd977caf90c787fcd10f8b0c642373b6466ecc4ae9acfdf8b002R460)...